packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -68,6 +66,64 @@ export interface ITree extends IFluidLoadable {
 	): TreeView<TRoot>;
 }
 
+/**
+ * A tree that may be {@link ViewableTree.viewWith | viewed} or {@link BranchableTree.branch | branched}.
+ * @alpha
+ */
+export interface BranchableTree extends ViewableTree {
+	/**
+	 * Create a new branch of the current state of the tree.
+	 * @remarks
+	 * The branch can be read and modified independently of the original tree.
+	 * A {@link TreeView | view} of the branch can be acquired using {@link ViewableTree.viewWith}().
+	 * Edits to the branch do not affect the original tree until they are merged back in.
+	 * Braches may also be rebased onto each other.
+	 */
+	branch(): TreeBranch;
+
+	/**
+	 * Apply all the new changes on the given branch to this tree.
+	 * @param branch - a branch which was created by a call to {@link BranchableTree.branch}().
+	 * @remarks It is automatically disposed after the merge completes.
+	 */
+	merge(branch: TreeBranch): void;
+
+	/**
+	 * Apply all the new changes on the given branch to this tree.
+	 * @param branch - a branch which was created by a call to {@link BranchableTree.branch}().
+	 * @param disposeView - whether or not to dispose `branch` after the merge completes.
+	 */
+	merge(branch: TreeBranch, disposeView: boolean): void;
+
+	/**
+	 * Rebase the changes that have been applied to the given branch over all the new changes in this tree.
+	 * @param branch - a branch which was created by a call to {@link BranchableTree.branch}().
+	 * It is modified by this operation.
+	 */
+	rebase(branch: TreeBranch): void;
+}
+
+/**
+ * Channel for a Fluid Tree DDS.
+ * @remarks
+ * Allows storing and collaboratively editing schema-aware hierarchial data.
+ * @sealed @public
+ */
+export interface ITree extends ViewableTree, IFluidLoadable {}
+
+/**
+ * A {@link BranchableTree.branch | fork} of a tree.
+ * @alpha
+ */
+export interface TreeBranch extends BranchableTree, IDisposable {
+	/**
+	 * Rebase the changes that have been applied to this view over all the new changes in the given view.
+	 * @param fork - Either the root tree or a tree that was created by a call to `fork()`.
```

Comment:
I don't see `fork()` in the public API. Should this be referencing `BranchableTree.branch`?

Reply:
Yes, thanks. All of this stuff was named "fork" and I renamed to "branch" and missed some.